### DISCUSSION TITLE
Question regarding send_text so simulate sending tmux prefix + other keys

### DISCUSSION THREAD

**Discussion by megalithic** [kovidgoyal/kitty#4812] [Q&A] [answered]
I had done this once before when first migrating to kitty years ago, but never documented it, and cannot seem to find out how to put this together.

I tried using kitty's `showkey` to watch the codes, but they don't necessarily translate to what i'd of expected.. here's an example of what I have presently for a tmux prefix of `C-a` (and the mapping i have in kitty config presently): `map ctrl+1 send_text all \x01\x31` -- this would send a key chord of `C+a>1`, however, I'd like to update this to `C+space>1`. I see that `\x01` is indeed `ctrl`, and `\x31` is indeed `1`, but I'm not sure how that translates to: `C+space>1`.

Any suggestions on how I can see what escaped unicode to put there for that "tmux passthrough" to work?

Thanks so much for kitty, since moving to it from iterm2, cli life has been wonderful.

**Comment by kovidgoyal** [comment, marked as answer]
Run

kitty +kitten show_key

press ctrl-space and then 1, you will see the output
UNIX		send_text
^@		\x00
1		1

which mean you need

\x001 

in send_text
